Machines/HPSS/arconfig_wrfout_fineIO_2.sb

```diff
@@ -0,0 +1,1006 @@
+    done # $DOMAINS
+  else
+    ## select datasets and logs
+    # logs and static data
+    if [[ "${DATASET}" == 'LOGS' ]] || [[ "${DATASET}" == 'DIAGS' ]] || [[ "${DATASET}" == 'FULL' ]]; then
+      local STATIC="static.tgz" # only on cold start, otherwise skip
+      for D in $( echo "${DOMAINS}" | fold --width=1 ); do
+	      STATIC="${STATIC} wrfconst_d0${D}.nc"
+      done # $DOMAINS
+      local LOGS="${DATE}*_pyWPS.tgz ${DATE}*_real.tgz ${DATE}*_wrf.tgz ${DATENAME}_logs.tgz"
+      # N.B.: log files have the step name and not the date in their file name, hence the irregularity
+    # diagnostics (one file per domain)
+    fi; if [[ "${DATASET}" == 'DIAGS' ]] || [[ "${DATASET}" == 'FULL' ]]; then      
+      DIAGS_D1="wrffdda_d01_${DATESTR}_??[:_]??[:_]??* wrfxtrm_d01_${DATESTR}_??[:_]??[:_]??* wrfsrfc_d01_${DATESTR}_??[:_]??[:_]??* wrfplev3d_d01_${DATESTR}_??[:_]??[:_]??* wrflsm_d01_${DATESTR}_??[:_]??[:_]??* wrfrad_d01_${DATESTR}_??[:_]??[:_]??* wrfhydro_d01_${DATESTR}_??[:_]??[:_]??*"
+      DIAGS_D2="wrfxtrm_d02_${DATESTR}_??[:_]??[:_]??* wrfsrfc_d02_${DATESTR}_??[:_]??[:_]??* wrfplev3d_d02_${DATESTR}_??[:_]??[:_]??* wrflsm_d02_${DATESTR}_??[:_]??[:_]??* wrfrad_d02_${DATESTR}_??[:_]??[:_]??* wrfhydro_d02_${DATESTR}_??[:_]??[:_]??*"  
+      DIAGS_D3="wrfxtrm_d03_${DATESTR}_??[:_]??[:_]??* wrfsrfc_d03_${DATESTR}_??[:_]??[:_]??* wrfplev3d_d03_${DATESTR}_??[:_]??[:_]??* wrflsm_d03_${DATESTR}_??[:_]??[:_]??* wrfrad_d03_${DATESTR}_??[:_]??[:_]??* wrfhydro_d03_${DATESTR}_??[:_]??[:_]??*"
+      DIAGS_D4="wrfxtrm_d04_${DATESTR}_??[:_]??[:_]??* wrfsrfc_d04_${DATESTR}_??[:_]??[:_]??* wrfplev3d_d04_${DATESTR}_??[:_]??[:_]??* wrflsm_d04_${DATESTR}_??[:_]??[:_]??* wrfrad_d04_${DATESTR}_??[:_]??[:_]??* wrfhydro_d04_${DATESTR}_??[:_]??[:_]??*"
+    ## select domain-wise datasets
+    # large 3D fields (one file per domain)
+    fi; if [[ "${DATASET}" == 'ATM3D' ]] || [[ "${DATASET}" == 'DEL3D' ]] || [[ "${DATASET}" == 'FULL' ]]; then
+      local ATM3D_D1="wrfdrydyn3d_d01_${DATESTR}_??[:_]??[:_]??* wrfmoist3d_d01_${DATESTR}_??[:_]??[:_]??* wrfrst_d01_${DATESTR}_??[:_]??[:_]??*"
+      local ATM3D_D2="wrfdrydyn3d_d02_${DATESTR}_??[:_]??[:_]??* wrfmoist3d_d02_${DATESTR}_??[:_]??[:_]??* wrfrst_d02_${DATESTR}_??[:_]??[:_]??*" 
+      local ATM3D_D3="wrfdrydyn3d_d03_${DATESTR}_??[:_]??[:_]??* wrfmoist3d_d03_${DATESTR}_??[:_]??[:_]??* wrfrst_d03_${DATESTR}_??[:_]??[:_]??*"
+      local ATM3D_D4="wrfdrydyn3d_d04_${DATESTR}_??[:_]??[:_]??* wrfmoist3d_d04_${DATESTR}_??[:_]??[:_]??* wrfrst_d04_${DATESTR}_??[:_]??[:_]??*"
+    # Snow files (one file per domain)
+    fi; if [[ "${DATASET}" == 'SNOW' ]] || [[ "${DATASET}" == 'DEL3D' ]] || [[ "${DATASET}" == 'FULL' ]]; then 
+      local SNOW_D1="wrfsnow_d01_${DATESTR}_??[:_]??[:_]??*"
+      local SNOW_D2="wrfsnow_d02_${DATESTR}_??[:_]??[:_]??*"
+      local SNOW_D3="wrfsnow_d03_${DATESTR}_??[:_]??[:_]??*"
+      local SNOW_D4="wrfsnow_d04_${DATESTR}_??[:_]??[:_]??*"
+    fi 
```

Comment:
We will likely add more options for retrieval, like only retrieving wrfsrfc and wrfplev3d or only wrfrst, but we can do that later, when we actually need it - it should be fairly straight forward.
I guess, if there is anything that you can think of, that we would need to keep in mind when doing that or that would make our future lives easier, it may help to add a comment or an example, but if there is nothing specific (I can't think of anything), we can leave it.